charts/langwatch/values.yaml

```
@@ -1,96 +1,1090 @@
+
+  # Upstream service connections
+  # Define how the main app connects to other LangWatch services
+  ## @extra app.upstreams Upstream connections from app to internal services.
+  ## @param app.upstreams.nlp.scheme [string, default: http] Scheme for NLP upstream.
+  ## @param app.upstreams.nlp.name [string, default: langwatch-nlp] Service name for NLP upstream.
+  ## @param app.upstreams.nlp.port [default: 5561] Port for NLP upstream.
+  ## @param app.upstreams.langevals.schema [string, default: http] Scheme for Langevals upstream.
+  ## @param app.upstreams.langevals.name [string, default: langevals] Service name for Langevals upstream.
+  ## @param app.upstreams.langevals.port [default: 5562] Port for Langevals upstream.
+  upstreams:
+    nlp:       { scheme: http, name: langwatch-nlp,  port: 5561 }
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)